src/study-access/studyAccessHooks.tsx

```diff
@@ -1421,19 +1431,21 @@ function makeContentSearchableSring(
     analysisPlan && 'Analysis Plan:',
     analysisPlan,
     disseminationPlan && 'Dissemination Plan:',
-    disseminationPlan
+    disseminationPlan,
+    priorAuth && 'Prior Authorization:',
```

Comment:
Does the value of `priorAuth` need to be added here?
```suggestion
    priorAuth && 'Prior Authorization:',
    priorAuth,
```